src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -56,47 +56,46 @@ static void cleanupHandle(uv_handle_t *handle) {
 }
 
 uv_check_t check_handle;
-uint32_t statistics[4];
-uint32_t& min = statistics[0];
-uint32_t& max = statistics[1];
-uint32_t& num = statistics[2];
-uint32_t& sum = statistics[3];
+int32_t min = 9999;
+int32_t max = 0;
+int32_t num = 0;
+int32_t sum = 0;
 
 
 #if NODE_VERSION_AT_LEAST(0, 11, 0) // > v0.11+
 static void GetLoopInformation(uv_timer_s *data) {
 #else
 static void GetLoopInformation(uv_timer_s *data, int status) {
 #endif
-	double mean = 0;
 	if (num != 0) {
+	  double mean = 0;
 		mean = sum / num;
-	}
 
-	std::stringstream contentss;
-	contentss << "NodeLoopData";
-	contentss << "," << min;
-	contentss << "," << max;
-	contentss << "," << num;
-	contentss << "," << mean;
-	contentss << '\n';
+	  std::stringstream contentss;
```

Comment:
You could remove all memory allocation by using a buffer on the stack, snprintfing into it with a single funtion call. it would also be one line of code, so easier to read, the multi-line iostream code seems pretty cumbersome to me in comparison.